apps/frontend/src/pages/auth/auth.tsx

```diff
@@ -48,10 +53,15 @@ const Auth: React.FC = () => {
 		return null;
 	};
 
+	if (user) {
+		<Navigate replace to={AppRoute.ROOT} />;
+	}
```

Comment:
```suggestion
	const hasUser = Boolean(user)
	if (hasUser) {
		return <Navigate replace to={AppRoute.ROOT} />;
	}
```
Make sure to test your code before pushing